Major aircraft / north american airline update

headphase opened the pull request:
= ADDED AIRCRAFT =

B712 - Boeing 717
B77L - Boeing 777 Freighter
B789 - Boeing 787-900
MD90 - Boeing MD-90
BN2P - Britten Norman Islander
C402 - Cessna 402
CRJ2 - CRJ200
CRJ7 - CRJ700
CRJ9 - CRJ900
DH8A - Dash 8 100
DH8C - Dash 8 300
E145 - Embraer RJ-145

= ADDED AIRLINES =

ASQ - Expressjet "acey"
BWA - Caribbean Airlines "caribbean airlines"
ENY - Envoy Air "envoy"
FLG - Endeavor Airlines "flagship"
GJS - GoJet Airlines "lindbergh
JBU - JetBlue "jetblue"
JIA - PSA airlines "blue streak"
KAP - Cape Air "cair"
PDT - Piedmont Airlines "piedmont"
RPA - Republic Airlines "brickyard"
TCF - Shuttle America "mercury"
SCX - Sun Country Airlines "sun country"
SWR - Swiss International Airlines "swiss"
VRD - Virgin America "redwood"

= UPDATED AIRLINES =

AAL 
AMX  (added "short" fleet type containing ERJs)
CPA
DAL
VIR
FDX

headphase committed:
Added/Update Airlines + Aircraft

ADDED AIRCRAFT

B712
CRJ2
CRJ9
E145
E175
MD90

ADDED AIRLINES

FLG
JBU
SCX
VRD

UPDATED AIRLINES

AAL
AMX
CPA
DAL

headphase committed:
file naming - e175

erikquinn commented on assets/aircraft/E175.json:
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Embraer 175",
+  "icao": "E175",
```
seems there are duplicates of this file... one with a lowercase `e175.json`, and this one with `E175.json`. Just so you know, the preferred convention is the lowercase one, so this one should be removed.

erikquinn commented on assets/aircraft/e175.json:
```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Embraer 175",
+  "icao": "E175",
```
I see you are adding the Embraer ERJ-175 in addition to the existing ERJ-170. The ICAO identifiers for those two aircraft are actually the same, because they are (essentially) the same model. Technically speaking, I think the 175 may be 10 feet longer (or less), and hold another two or so rows of passengers. Thus, ICAO treats them as the same model, as does the FAA:

![image](https://cloud.githubusercontent.com/assets/5103735/14430885/db5ff3bc-ffc8-11e5-99db-bc5c28061021.png)
![image](https://cloud.githubusercontent.com/assets/5103735/14430985/47433724-ffc9-11e5-857f-dac495e8289d.png)

The best course then, I think, would be just to leave both in the `e170.json` file only, under the icao identifer of `E170`.

erikquinn commented:
At first glance, all looks good other than the E170/E175 situation mentioned above. Once we get that squared away, I'll be glad to finally see some E-Jets and CRJ's around here. It's been too long without them! :laughing:

headphase committed:
removed e175

headphase commented:
Aha!  My bad; that's what I get for using wiki; I was going off of [this page](https://en.wikipedia.org/wiki/List_of_ICAO_aircraft_type_designators) but I guess I never learned my lesson from all those high school teachers years ago haha.

I've removed the e175 completely and updated AMX to reflect it.

headphase committed:
Delete E175.json

headphase committed:
Delete e175.json

headphase committed:
Update e170.json

erikquinn commented:
@headphase haha!

Cool, I ran it through just now and if you can just remove the call to load "e175" from `aircraft.js : 2516`, then we'll be all set here.

headphase committed:
delete e175 from aircraft.js

headphase committed:
MORE aircraft + airlines

headphase committed:
fixed DH-8A/C mach speed
